steering_engine_hw:
  serial:
    name:
      - "/dev/usbReferee"